cmd/dockerd/daemon_unix.go

```diff
@@ -107,3 +109,18 @@ func wrapListeners(proto string, ls []net.Listener) []net.Listener {
 	}
 	return ls
 }
+
+func newCgroupParent(config *config.Config, d *daemon.Daemon, sm *session.Manager) string {
```

Comment:
why are daemon and sessionmanager passed to this function

Reply:
because I forgot to remove them 😓